### DISCUSSION TITLE
Appointment view as Outlook or similar

### DISCUSSION THREAD

**Discussion by msavazzi** [MahApps/MahApps.Metro#4047] [Ideas]
Team,
I've been searching a lot for a decent calendar view, I'm used (as I think everyone) to outlook kind of experience.
I've not found any solution apart from 900$ and above :( and for a personal project is not really feasible... also not sure how they would work together with mahapps.

Will it be possible to add it? I'll be happy to help, if needed, with the logic behind the control but I'm awful, terrible in XAML and design.

I think it would be an amazing control to add :)

any thought ?

**Comment by makingbloke** [comment]
The Syncfusion WPF controls are free for personal use / teams smaller than 5. It includes a diary control that may do what you want. (In my own project I use MahApps for the bulk of the GUI work along with some custom controls and Syncfusion for their Office document tools.)

https://www.syncfusion.com/wpf-ui-controls/scheduler
